packages/material-ui/src/Button/Button.js

```diff
@@ -333,10 +285,6 @@ Button.propTypes = {
    * If defined, an `a` element will be used as the root node.
    */
   href: PropTypes.string,
-  /**
-   * If `true`, and `variant` is `'fab'`, will use mini floating action button styling.
-   */
-  mini: PropTypes.bool,
```

Comment:
True, although it was only useful with variant="fab", so anyone using it should be getting variant deprecation warnings. It would probably have been a bit redundant and noisy to warn for both.

Reply:
True. I'm glad we removed this here. It's rarely a good idea to allow invalid props combinations.